js/actions/windowActions.js

```diff
@@ -92,6 +92,24 @@ const WindowActions = {
   },
 
   /**
+   * Similar to loadUrl, but loads the URL in a specified frame instead of the
+   * active one.
+   * @param {string} location - url to load
+   * @param {number} key - key of the frame to load in
+   */
+  loadUrlInFrame: function (location, key) {
```

Comment:
Instead of this please modify loadUrl, it currently takes in a frameProps already but it doesn't pass the key to the store.  It would be better for it to pass the key to the store and for the store to use it.